src/cheri/cheri-pte-ext.adoc

```diff
@@ -196,6 +196,16 @@ When V=1 <<vsstatusreg_pte,vsstatus>>.UCRG and CRGE are in effect.
 
 <<mstatusreg_pte,mstatus>>.UCRG and CRGE also exist. Reading or writing it is equivalent to reading or writing <<sstatusreg_pte,sstatus>>.UCRG and CRGE respectively.
 
+NOTE: The CRGE bit is added to the <<mstatusreg_pte,mstatus>>, <<vsstatusreg_pte,vsstatus>> and <<sstatusreg_pte,sstatus>> registers in order to allow the S-mode software to explicitly opt in to the revocation extensions.
+The CRGE bit can not be exclusively accessible from M-mode (e.g. in <<menvcfg>>), because this would imply that M-mode can enable revocation without any guarantee that the S-mode software understands the CRW, CD and CRG semantics, and can result in breaking backward compatibility when updating the M-mode firmware.
```

Comment:
I'm not sure that this last sentence is really relevant... M mode does a lot of things at the behest of less-privileged modes without acting of its own volition.  In principle, turning on `Svucrg` could be done by some non-architectural communication path through the hypervisor and to M mode (SBI/ecall/...).

In any case, the first sentence of this NOTE is fine.

Reply:
Yeah I mean you *can* do it in M-mode, that's where ADUE lives, but it's stupid, because it requires an SBI call to do something that S-mode is in control of (it's always S-mode's choice which value to use, and there's no reason for M-mode to deny it) and has no implications for M-mode. We should not repeat that idiocy and just make it available to S-mode.

Reply:
Changed the wording now.